src/applications/personalization/profile-2/sass/mobile-sidenav-trigger.scss

```diff
@@ -23,4 +26,7 @@
       width: auto;
     }
   }
+  @include media($medium-screen) {
```

Comment:
this was previously getting set by sitewide styles applied to the `va-btn-sidebarnav-trigger` class